forui/lib/src/widgets/tabs/tabs.dart

```diff
@@ -72,6 +72,9 @@ class FTabs extends StatefulWidget {
   /// Handler for when a tab is changed.
   final ValueChanged<int>? onChange;
 
+  /// Triggered when the tab is clicked
+  final ValueChanged<int>? onTap;
```

Comment:
nitpick: I think `onPress` is a better name. It'll make it more consistent with our current API.